main/midnight/src/system/resolutionmanager.h

```diff
@@ -76,6 +76,8 @@ class resolutionmanager
 };
 
 #define RES(x) (f32)((x)*resolutionmanager::getInstance()->Scale())
+// FJAR: what is DIS?
```

Comment:
I added 'DIS' macro to have it compile. Not clear what DIS should be. Can you explain what this should be?

Reply:
It should just be RES. I must have missed some places. At some stage I had a difference between RES and DIS - possibly font related. But not needed anymore.